core/src/main/scala/dagr/core/cmdline/parsing/CommandLineParser.scala

```diff
@@ -62,9 +58,9 @@ object ParseResult extends Enumeration {
   val Version = Value // A request to display version was encountered
 }
 
-object CommandLineParser {
-  private[parsing] def parseArgumentsFile(lines: Traversable[String]): Traversable[String] = {
-    val args: ListBuffer[String] = new ListBuffer[String]
+private[parsing] object CommandLineParser {
+  def parseArgumentsFile(lines: Traversable[String]): Traversable[String] = {
+    val args: mutable.ListBuffer[String] = new mutable.ListBuffer[String]
```

Comment:
I'm just curious why this change?  There's no immutable ListBuffer so it was non-ambiguous before.